src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1577,7 +1577,7 @@ limitations under the License.
                 <requireCommand>RemoveAllGroups</requireCommand>
                 <requireCommand>AddGroupIfIdentifying</requireCommand>
             </include>
-            <include cluster="Scenes" client="true" server="true" clientLocked="true" serverLocked="true">
+            <include cluster="Scenes" client="true" server="false" clientLocked="true" serverLocked="true">
```

Comment:
Where is this coming from?  I see nothing in the Matter device library with an M client Scenes cluster.  This applies to a bunch of other cases in this PR too.